test/ExportVerilog/sv-verifLabels.mlir

```diff
@@ -0,0 +1,26 @@
+// RUN: circt-translate --lowering-options= --export-verilog %s | FileCheck %s --check-prefix=CHECK-OFF
+// RUN: circt-translate --lowering-options=verifLabels --export-verilog %s | FileCheck %s --check-prefix=CHECK-ON
+
+hw.module @foo(%clock: i1, %cond: i1) {
+  sv.initial {
+    // CHECK-OFF: assert(cond);
```

Comment:
It is worth making sure that explicitly labeled sv.assert maintain their label.  Even more so, that if an explicit label conflicts with an auto-generated one, the auto-generated one is the one that changes.

Reply:
Good point. Let me add that to the test.